10.js

```diff
@@ -0,0 +1,23 @@
+const resolveSymlinks = input => {
+  const inputArray = input.split('\n')
+  let symlinks = []
+  let path = inputArray[inputArray.length - 1]
+
+  for (let index = 1; index <= inputArray[0]; index++) {
```

Comment:
Here's the `temp collection variable, iterate and mutate temp collection variable, return temp collection variable` pattern that I tend to replace with a more functional solution (`map`, in this case):
```js
const paths = input.split( '\n' )
let path = paths.pop()

const symlinks = paths.map( path => path.split( ':' ) )
```